src/sentry/api/endpoints/group_external_issues.py

```diff
@@ -1,13 +1,23 @@
+import logging
+
 from sentry.api.bases.group import GroupEndpoint
 from sentry.api.serializers import serialize
 from sentry.models import PlatformExternalIssue
 
+logger = logging.getLogger("sentry.api")
+
 
 class GroupExternalIssuesEndpoint(GroupEndpoint):
     def get(self, request, group):
 
         external_issues = PlatformExternalIssue.objects.filter(group_id=group.id)
 
+        logger.info(
+            "group_external_issue.get",
+            extra={
+                "user": request.user,
```

Comment:
is the `request.user` a whole object? I feel like maybe we can have a `is_sentry_app_user` and have that be logged in addition to maybe the `user.username` if it's there?

Reply:
@MeredithAnya how do you feel about the email rather than the username? I think that'd be potentially easier to look up which org it is.

Reply:
@ceorourke so im pretty sure we set the `user.username` to be the sentry_app slug:

https://github.com/getsentry/sentry/blob/c2d66633ce304f9d96b0bfb850662d79e4a2a75c/src/sentry/mediators/sentry_apps/creator.py#L62

which is why I suggested it. Since I think we really only care about lookin up the org if it's an api token that's been used right?

Reply:
ahh okay, yeah that makes sense